boards/nucleo-l073/include/periph_conf.h

```diff
@@ -175,10 +175,16 @@ static const spi_conf_t spi_config[] = {
 /** @} */
 
 /**
- * @name    ADC configuration
+ * @name   ADC configuration
```

Comment:
This change is not 'valid', we want to keep 4 spaces after `@name`

Reply:
done